Aquifer/NSFileHandleExt.m

```diff
@@ -50,9 +45,9 @@ + (instancetype)aqu_fileHandleWithConnectionToHost:(NSString *)host toPort:(int)
 
 	struct sockaddr_in remoteaddr;
 	bzero((char *)&remoteaddr, sizeof(remoteaddr));
-	remoteaddr.sin_family = AF_INET;
+	remoteaddr.sin_family		= AF_INET;
```

Comment:
Ah!  They're aligned with the previous/following assignments.  I thought it looked good, but I can disable that of you want.